release-notes/major9/09.19.0/valtimo-frontend-libraries.md

```diff
@@ -13,6 +13,33 @@ The following features were added:
 
   Various tooltips for plugin configuration components have been added or improved.
 
+* **Improved the select component of the @valtimo/user-interface library**
+  * **Background color change**
+
+    The select component has been improved to follow the default input style and make it visibly indicated when a 
```

Comment:
The select component has been improved to follow the default input style and to visually indicate when a dropdown menu is enabled or disabled.